GitBook: [dx_add-own-connectors] 2 pages modified (#693)
airbytehq/airbyte: docs/integrations/adding-your-own-connectors.md

> # Adding your own connectors

Reviewer: Should this be renamed `Custom Connectors`? It's less confusing if we can standardize how we refer to features.

Author: `Custom Connectors` is way better indeed. Will make the change!

Author: Committed the changes

If you'd like to build new connectors and make them part of the pool of pre-built connectors on Airbyte, first a big thank you, and we invite you to check our [contributing guide](https://docs.airbyte.io/contributing/contributing-to-airbyte).

> If you'd like to build new connectors, or update existing ones, for your own usage, without contributing to the Airbyte codebase, then that's what open-source is for, and read along. 

Reviewer: ```suggestion
If you'd like to build new connectors, or update existing ones, for your own usage, without contributing to the Airbyte codebase, read along. 
```

2. [Developing your own connectors](https://docs.airbyte.io/v/dx-to-add-own-connector_suggestion/integrations/adding-your-own-connectors#2-developing-your-own-connectors)
3. [Adding your connectors on our UI to run them](https://docs.airbyte.io/v/dx-to-add-own-connector_suggestion/integrations/adding-your-own-connectors#3-adding-your-connectors-on-our-ui-to-run-them)

## 1. The Airbyte specification

Before you can start building your own connector, you need to understand [Airbyte's data protocol specification](https://docs.airbyte.io/architecture/airbyte-specification). 

## 2. Developing your own connectors

It's easy to code your own integrations on Airbyte. Here are some links to instruct on how to code new sources and destinations. 

#### **Coding new source connectors:**

* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)

> #### **Coding new destination connectors:**

Reviewer: There should also be a step before the UI section that says that in order for Airbyte to be able to leverage a custom integration, it should be published to Dockerhub first (or any image hub that Airbyte can access).

Author: I just added a step in section 3 for that, as a first step. Let me know if this is good for you.

* [In Java](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/java-destination/README.md)

> Please reach out to us if you'd like help developing integrations in other languages. **Indeed, you are able to create integrations using the language you want, as Airbyte connections run as Docker containers.** We haven't built the documentation for all languages yet, that's all. 

Reviewer: ```suggestion
While the guides above are specific to the languages use most frequently to write integrations, **Airbyte integrations can be written in any language**. Please reach out to us if you'd like help developing integrations in other languages.
```

> ## 3. Adding your connectors on our UI to run them

Reviewer: ```suggestion
## 3. Adding your connectors in the UI
```
